tests/Unit/Http/Client/CircuitBreaker/CircuitBreakerTest.php

```diff
@@ -72,7 +73,7 @@ public function itShouldOpenCircuitOnThreshold()
 
         for ($i = 0; $i <= $circuitBreaker->getThreshold(); ++$i) {
             $response = $circuitBreaker->call(function () {
-                throw new ConnectException('Test Timeout Reached', $this->getRequest());
+                throw new ConnectException('Test Timeout Reached');
```

Comment:
Not useful now to pass the request to the exception ?

Reply:
Probably useless yes